app/grandchallenge/pages/templates/pages/page_content_update.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{% load crispy from crispy_forms_tags %}
+{% load static %}
+{% load url %}
+
+{% block title %}
+    {% if object %}Update{% else %}Create{% endif %} -{% if object %} {{ object|title }} -{% endif %} {{ block.super }}
```

Comment:
Do the title and breadcrumbs make sense for this template?

Reply:
Thank you. Removed.